doc/source/user_guide/io.rst

```diff
@@ -5477,6 +5478,42 @@ web site.
 
 No official documentation is available for the SAS7BDAT format.
 
+.. _io.spss:
+
+.. _io.spss_reader:
+
+SPSS formats
+------------
+
+The top-level function :func:`read_spss` can read (but not write) SPSS
+`sav` (.sav) and  `zsav` (.zsav) format files(since *v0.25.0*).
```

Comment:
Rather than saying since 0.25 here you can add a versionadded directive before this for 0.25 (check docs for other examples)